toolchain/check/pattern_match.h

```diff
@@ -10,15 +10,14 @@
 
 namespace Carbon::Check {
 
-// The pattern-match counterparts of the patterns passed to CalleePatternMatch.
+// The outputs of CalleePatternMatch.
+// TODO: rename or remove this struct.
 struct ParameterBlocks {
-  // The implicit parameter list.
-  SemIR::InstBlockId implicit_params_id;
-
-  // The explicit parameter list.
-  SemIR::InstBlockId params_id;
+  // The call parameters of the function.
+  SemIR::InstBlockId call_params_id;
 
   // The return slot.
+  // TODO: drop this and just use the last element of above?
```

Comment:
```suggestion
  // TODO: Drop this and just use the last element of above?
```

(see style comment above)